### DISCUSSION TITLE
Pause all video's should pause live telemetry too

### DISCUSSION THREAD

**Discussion by spambas** [f1multiviewer/issue-tracker#123] [Feature requests] [answered]
I shift+space and every video paused (great feature) and when I came back hit shift+space and it started again. After that I could not figure out what happened: **the live timing did not pause.** 

So I was watching the OBC's with 1,5 lap newer live timing. Certainly during this race where the F1TV data channel crashed multiple times...

**Comment by jeppeolesen** [comment, marked as answer]
This has been implemented in one of the many updates since 2022 😅
